plugins/azure_sentinel/plugin.spec.yaml

```diff
@@ -0,0 +1,879 @@
+    classification:
+      description: The reason the incident was closed
+      title: The reason the incident was closed
+      type:  string
+      enum:
+        - BenignPositive
+        - FalsePositive
+        - TruePositive
+        - Undetermined
+    classificationComment:
+      description: Describes the reason the incident was closed
+      title: Describes the reason the incident was closed
+      type: string
+    classificationReason:
+      description: The classification reason the incident was closed with
+      title: The classification reason the incident was closed with
+      type: string
+      enum:
+        - InaccurateData 
+        - IncorrectAlertLogic
+        - SuspiciousActivity
+        - SuspiciousButExpected
+    createdTimeUtc:
+      description: The time the incident was created
+      title: The time the incident was created
+      type: date
+    description:
+      description: The description of the incident
+      title: The description of the incident
+      type: string
+    etag:
+      description: Etag of the azure resource
+      title: Etag of the azure resource
+      type: string
+    firstActivityTimeUtc:
+      description: The time of the first activity in the incident
+      title: The time of the first activity in the incident
+      type: date
+    id:
+      description: Azure resource Id
+      title: Azure resource Id
+      type: string
+    incidentNumber:
+      description: A sequential number
+      title: A sequential number
+      type: integer
+    incidentUrl:
+      description: The deep-link url to the incident in Azure portal
+      title: The deep-link url to the incident in Azure portal
+      type: string
+    labels:
+      description: List of labels relevant to this incident
+      title: List of labels relevant to this incident
+      type: "[]IncidentLabel"
+    lastActivityTimeUtc:
+      description: The time of the last activity in the incident
+      title: The time of the last activity in the incident
+      type: date
+    lastModifiedTimeUtc:
+      description: The last time the incident was updated
+      title: The last time the incident was updated
+      type: date
+    name:
+      description: Azure resource name
+      title: Azure resource name
+      type: string
+    owner:
+      description: Describes a user that the incident is assigned to
+      title: Describes a user that the incident is assigned to
+      type: IncidentOwnerInfo
+    relatedAnalyticRuleIds:
+      description: List of resource ids of Analytic rules related to the incident
+      title: List of resource ids of Analytic rules related to the incident
+      type: "[]string"
+    severity:
+      title: Incidents severity
+      description: Incidents severity
+      required: true
+      type: string
+      enum:
+        - High
+        - Informational
+        - Low
+        - Medium
+    status:
+      title: Incidents status
+      description: Incidents status
```

Comment:
Description should not be same as title

Reply:
@igorski-r7 for a custom output type it is totally fine. Usually there is no description given by the API vendor, so we don't have much choice 😓  For an action or input description, however, they should be different. Good callout though!